static/js/components/PostDisplay.js

```diff
@@ -69,28 +70,33 @@ class PostDisplay extends React.Component {
     })
   }
 
-  render() {
+  upvoteDisplay = () => {
     const { post, expanded } = this.props
     const { upvoting } = this.state
-    const formattedDate = moment(post.created).fromNow()
     const upvoteClass = post.upvoted ? "upvoted" : ""
+
     return (
-      <div className="post-summary">
-        <div className={`upvotes ${upvoteClass}`}>
-          <button
-            className="upvote-button"
-            onClick={this.onToggleUpvote}
-            disabled={upvoting}
-          >
-            <img
-              className="upvote-arrow"
-              src="/static/images/upvote_arrow.png"
-            />
-          </button>
-          <span className="votes">
-            {post.score}
-          </span>
-        </div>
+      <div className={`upvotes ${upvoteClass} ${expanded ? "expanded" : ""}`}>
```

Comment:
What does the CSS class `expanded` do here?

Reply:
basically I'm just using it so I can choose between `flex-direction: row` and `flex-direction: column`

Reply:
b/c when we're showing a post in a `PostList` (i.e. in an index list of posts) the voting buttons appear in a column, but on the post page they appear in a row.